static/js/admin.js

```diff
@@ -50,6 +50,30 @@ exports.update_user_data = function (user_id, new_data) {
         user_row.find(".owner").text(new_data.owner);
     }
 
+    if (new_data.is_active !== undefined) {
+        if (new_data.is_active === false) {
+            // Deactivate the bot in the table
+            var deactivate_button = user_row.find("button.deactivate");
+            user_row.find('button.open-user-form').hide();
+            deactivate_button.addClass("btn-warning");
+            deactivate_button.removeClass("btn-danger");
+            deactivate_button.addClass("reactivate");
+            deactivate_button.removeClass("deactivate");
+            deactivate_button.text(i18n.t("Reactivate"));
+            user_row.addClass("deactivated_user");
+        } else {
+            // Reactivate the bot in the table
+            var reactivate_button = user_row.find("button.reactivate");
+            user_row.find("button.open-user-form").show();
+            reactivate_button.addClass("btn-danger");
+            reactivate_button.removeClass("btn-warning");
+            reactivate_button.addClass("deactivate");
+            reactivate_button.removeClass("reactivate");
+            reactivate_button.text(i18n.t("Deactivate"));
+            user_row.removeClass("deactivated_user");
```

Comment:
What I'd personally do is actually rebase and add a first commit that extracts the logic, and then simplify this commit to just need to call the function and do any extra details.    Could be a good rebase practice opportunity for you :)